src/components/LightsOut.tsx

```
@@ -0,0 +1,146 @@
+
+  const isWon = grid.flat().every((cell) => !cell);
+
+  if (!mounted) return null;
+
+  return (
+    <div className={styles.container}>
+      <h1>Lights Out</h1>
+
+      <div className={styles.controls}>
+        <span>Difficulty:</span>
+        <button className={styles.difficultyButton} onClick={() => setGridSize(3)}>Easy</button>
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)